debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9ObjectHelper.java

```diff
@@ -68,27 +89,44 @@ public static boolean isIndexable(J9ObjectPointer objPointer) throws CorruptData
 	public static U32 flags(J9ObjectPointer objPointer) throws CorruptDataException
 	{		
 		if(J9BuildFlags.interp_flagsInClassSlot) {
```

Comment:
Since you removed the test of `interp_flagsInClassSlot`, perhaps you should remove it here too?

Reply:
Done.